test/runtests.jl

```diff
@@ -20,18 +20,18 @@ thermo_params = SFP.thermodynamics_params(param_set)
 uft = SFP.universal_func_type(param_set)
 
 const TD = Thermodynamics
+device(::T) where {T <: Array} = CPU()
```

Comment:
Thanks @nefrathenrici This will get an overhaul in the next couple of weeks as we start to move towards enabling GPU tests.